style.css

```diff
@@ -454,12 +453,10 @@ textarea:focus {
 
 .common-li {
   color: #fff5e1;
-  font-size: 32px;
-  padding-bottom: 16px;
+  font-size: 23px;
 
   /* text-align: center; */
   margin: auto;
-  border-bottom: solid 1px #6f6c6b;
 }
```

Comment:
- [ ] Kindly try to use the same font sizes that are provided in the Figam template so your UI design is better.

<img width="724" alt="image" src="https://user-images.githubusercontent.com/77190203/155486008-3d9c8d09-640b-44c7-bef9-01fb57676380.png">